src/components/PersonListItem/PersonListItem.tsx

```diff
@@ -0,0 +1,55 @@
+import { Card, CardActionArea, CardContent, Typography } from '@mui/material'
+import { ReactElement } from 'react'
+
+import { GetContacts } from '../../lib/queries/getContacts/getContacts'
+import { GetProcessDefinitions } from '../../lib/queries/getProcessDefinitions/getProcessDefinitions'
+
+interface PersonListItemProps {
+  contact: GetContacts
+  definitions: GetProcessDefinitions
+}
+
+export function PersonListItem({
+  contact,
+  definitions
+}: PersonListItemProps): ReactElement {
+  const processKeys = Object.keys(contact.process)
+  const processes = processKeys.map((key) => {
+    return definitions[key].title
+  })
+  const processLabel =
+    processes.length === 0
+      ? ''
+      : processes.length === 1
+      ? processes[0]
+      : processes.length === 2
+      ? `${processes.slice(0, -1).join(', ')} & ${processes.slice(-1)}`
+      : `${processes.slice(0, 2).join(', ')} & ${
+          processes.slice(2).length
+        } more`
+
+  let taskCount = 0
+  Object.values(contact.process).forEach((process) => {
+    taskCount += process.taskCount.incomplete + process.taskCount.pending
+  })
+
+  const taskLabel =
+    taskCount === 1 ? `${taskCount} faith step` : `${taskCount} faith steps`
+
+  return (
+    <Card sx={{ width: '100%', maxWidth: 345, mb: 2 }}>
+      <CardActionArea href={`/tabs/people/${contact._id}`}>
+        <CardContent>
+          <Typography variant="caption" color="text.secondary">
+            {processLabel}
+          </Typography>
+          <Typography
+            variant="h6"
+            gutterBottom
+          >{`${contact.firstName} ${contact.lastName}`}</Typography>
```

Comment:
Don't think you need to have a special variable js string here.
```suggestion
          <Typography variant="h6" gutterBottom>
            {contact.firstName} {contact.lastName}
          </Typography>
```